services/analyzer/sequelize-tables-analyzer.js

```diff
@@ -371,6 +371,12 @@ async function analyzeSequelizeTables(databaseConnection, config, allowWarning)
   const referencesPerTable = createAllReferences(databaseSchema, schemaAllTables);
   Object.keys(referencesPerTable).forEach((tableName) => {
     schemaAllTables[tableName].references = _.sortBy(referencesPerTable[tableName], 'association');
+
+    if (!schemaAllTables[tableName].fields.length) {
+      schemaAllTables[tableName].options.underscored = isUnderscored(
+        schemaAllTables[tableName].references.map(({ foreignKey }) => ({ nameColumn: foreignKey })),
+      );
```

Comment:
Not sure to understand the logic here.
I wonder if this is another fix (vs the `computedReference.junctionTable` in the file below)

Reply:
Technically this is another fix, but both are related to the associated ClickUp task.

When a table contains no fields, `options.underscored` will always be `false` because it's computed in the fields analysis process (before references checks). That doesn't mean that the table is not snake_cased though, as it's possible that there are FKs that  are actually snake_cased.

The ideal solution would be to loop through schemaAllTables only when fields AND references are analyzed, but that would lead to a performance impact.